sklearn/manifold/tests/test_t_sne.py

```diff
@@ -247,6 +247,15 @@ def test_trustworthiness():
     assert_almost_equal(trustworthiness(X, X_embedded, n_neighbors=1), 0.2)
 
 
```

Comment:
Can you please add a non-regression test for https://github.com/scikit-learn/scikit-learn/issues/18567 based on [this comment](https://github.com/scikit-learn/scikit-learn/issues/18567#issue-716945099) snippet?